public/js/components/Header/Header.js

```diff
@@ -82,6 +82,19 @@ class Header extends React.Component {
     return false;
   }
 
+  renderCreateNewButton = () => {
+    if(location.pathname === "/list") {
```

Comment:
Can we not pull this from `location`? There should be a `this.props.router` with a route on it. That's the actual source of truth with what route we're currently on (not the URL bar...)